src/diracx/cli/__init__.py

```diff
@@ -22,8 +22,8 @@
 @app.async_command()
 async def login(
     vo: str,
-    group: Optional[str] = None,
-    property: Optional[list[str]] = Option(
+    group: Optional[str] = None,  # noqa: UP007
```

Comment:
Shall we do this at the file level with a comment explaining why?

```
# ruff: noqa: F841
```

https://beta.ruff.rs/docs/configuration/#error-suppression